extensions/sql-migration/package.json

```diff
@@ -10,7 +10,7 @@
   "aiKey": "29a207bb14f84905966a8f22524cb730-25407f35-11b6-4d4e-8114-ab9e843cb52f-7380",
   "engines": {
     "vscode": "*",
-    "azdata": ">=1.41.0"
+    "azdata": ">=1.42.0"
```

Comment:
this should stay 1.41, that's the version that's about to come out. I think Akshay already bumped this recently

Reply:
yeah, @AkshayMata did this just yesterday https://github.com/microsoft/azuredatastudio/pull/21711

Reply:
Fixed. Thanks